src/site/topic22.rst

```diff
@@ -202,11 +202,53 @@ Traversal Analysis
 
 * :math:`O(2^{h})`
 * Why?
-* Consider the relationship between the height of a binary tree and the number of nodes within the tree  
+* Consider the relationship between the height of a binary tree and the number of nodes within the tree
+
 
 Interface
 =========
 
+* What would we want to do with our binary trees?
```

Comment:
Presumably branched from the wrong place? Revert this.

Reply:
Yerp